core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -3628,6 +3635,7 @@ private static String javadocLink(String fullyQualifiedClassname) {
     public static final String INTEGRATION_YARN_WORKERS_PER_HOST_MAX =
         "alluxio.integration.yarn.workers.per.host.max";
     public static final String LOGGER_TYPE = "alluxio.logger.type";
+    public static final String USER_LOGS_DIR = "alluxio.user.logs.dir";
```

Comment:
nit: I think we try to keep these in alphabetical order

Reply:
done.